src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -275,12 +275,11 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_do_irecv(void *buf,
             chunk_req->event_id = MPIDI_OFI_EVENT_RECV_GPU_PIPELINE_INIT;
             chunk_req->parent = rreq;
             chunk_req->buf = host_buf;
-            int ret = 0;
-            ret = fi_trecv(MPIDI_OFI_global.ctx[ctx_idx].rx,
-                           host_buf,
-                           MPIR_CVAR_CH4_OFI_GPU_PIPELINE_BUFFER_SZ,
-                           NULL, remote_addr, match_bits, mask_bits,
-                           (void *) &chunk_req->context);
+            mpi_errno = fi_trecv(MPIDI_OFI_global.ctx[ctx_idx].rx,
```

Comment:
Use `MPIDI_OFI_CALL`?

Reply:
Maybe. Actually, Gengbin changed into this from `MPIDI_OFI_CALL` not long ago to avoid the recursive event triggering issue. Anyway, I would prefer to touch the pipeline code as little as we can for now, considering the plan is to revamp this soon.

Reply:
It looks like this PR actually undoes Gengbin's fix and introduces this warning. See 416f3ef5a4d9d08281b7cc9d215c8f4ccc17c80b.

Reply:
Not sure what happens here regarding to the commit 416f3ef. I think calling MPIDI_OFI_CALL is fine to replace MPIDI_OFI_CALL_RETRY.

Reply:
Good catch! I'll fix it.

Reply:
Fixed. The warnings fix no longer needed! :)